### DISCUSSION TITLE
centos7.9 binary install docker,systemctl start docker stuck,disable selinux start success

### DISCUSSION THREAD

**Discussion by felix-186** [moby/moby#44441] [Q&A]
### Description

OS: centos7.9，binary install docker, systemctl start docker stuck, kill failure, setenforce 0, systemctl start docker is ok, start selinux is not ok.

### Reproduce

 systemctl start docker stuck

### docker version

```bash
19.03.15
```

### docker info

```bash
Client:
 Debug Mode: false

Server:
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
errors pretty printing info
```

**Comment by thaJeztah** [comment]
Are you still seeing this issue on a current version (20.10 or up) of docker?

**Comment by felix-186** [comment]
> Are you still seeing this issue on a current version (20.10 or up) of docker?

20.10 has the same problem

**Comment by bsousaa** [comment]
We couldn't reproduce this issue. We are converting the issue to a Discussion to reach more people in the community.
